hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/io/ECKeyOutputStream.java

```diff
@@ -154,7 +154,7 @@ public void write(byte[] b, int off, int len) throws IOException {
     }
     try {
       int writtenLen = 0;
-      while (off + writtenLen < len) {
```

Comment:
Thanks for fixing it @swamirishi. This needs a test.

Reply:
@kaijchen do you want to take a look at it?

Fortunately most of the apps seems to be using off starts from 0, so, we have not hit this one :-)

Reply:
LGTM. Thanks for the fix @swamirishi.